src/Avalonia.Controls/SelectableTextBlock.cs

```diff
@@ -30,7 +30,7 @@ public class SelectableTextBlock : TextBlock, IInlineHost
                 o => o.SelectedText);
 
         public static readonly StyledProperty<IBrush?> SelectionBrushProperty =
-            TextBox.SelectionBrushProperty.AddOwner<SelectableTextBlock>(new(new Data.Optional<IBrush?>(Brushes.Blue)));
+            TextBox.SelectionBrushProperty.AddOwner<SelectableTextBlock>();
```

Comment:
TextBox.SelectionBrushProperty doesn't have a default value, so I removed it from there as well.